addons/mrp/models/mrp_bom.py

```diff
@@ -104,6 +106,11 @@ def _compute_possible_product_template_attribute_value_ids(self):
         for bom in self:
             bom.possible_product_template_attribute_value_ids = bom.product_tmpl_id.valid_product_template_attribute_line_ids._without_no_variant_attributes().product_template_value_ids._only_active()
 
+    @api.depends('company_id')
+    def _compute_currency_id(self):
+        for company in self:
```

Comment:
for company in self. I'm not sure about that...

Reply:
Done.